src/pages/experiments/[experimentId]/data-processing/data-integration/components/CalculationConfig.jsx

```diff
@@ -97,7 +96,7 @@ const CalculationConfig = () => {
         {changesOutstanding && (
           <Form.Item>
             <Alert
-              message='Your changes are not yet applied. To update the plots, click Apply.'
+              message='Your changes are not yet applied. To re run data integration, click Apply.'
```

Comment:
usability issue: this message does not trigger after I change the numbers for `# of genes` or the `number of principal components` -- in order for the alert to appear, I actually have to click on the screen itself.
Also, if I change a setting and then unchange it, the message that my settings are not applied dos not appear, which is not accurate. Now I understand that this is a hard one to fix, but if we don't fix it here on the UI side, it would cause unnecessary requests to the backend and unnecessary reruns of the future step functions pipeline, which will be an even bigger pain.

Reply:
I also actually, just fyi, no actions for this specific change, but because of this issue I just realised that we might need to disable the caches for data processing entirely because otherwise users will have discrepancies in their count matrix files if they revert back to previously selected settings (I added a comment here: https://docs.google.com/document/d/1cVqHMt9gqyS3vYnCdhcZrKeokXDAQDlwlCF5lmNYf8w/edit#heading=h.3etmldd107r6).

Reply:
This only happens when you type in the number right?

Reply:
Fixed in latest commit

Reply:
yup, only when I type in the number

Reply:
I don't think your comment will be an issue -- added a comment to the google doc @ivababukova

Reply:
thanks Marcell, I will check there and respond there directly

Reply:
Can you change `re run` to `rerun`?